### ISSUE TITLE
BASE_DIR error

### ISSUE THREAD

**Issue by casoetan** [ImperialCollegeLondon/django-drf-filepond#5]
Line 20 - 21 in this file (which is used for testing)
```
django_drf_filepond/drf_filepond_settings.py
```
```
UPLOAD_TMP = getattr(settings, _app_prefix+'UPLOAD_TMP',
                     os.path.join(settings.BASE_DIR,'filepond_uploads'))
```

requires `BASE_DIR` be set but my config is customized and I don't use that.

I don't think users should be forced to set the `BASE_DIR` if they do not need it.

**Comment by jcohen02** [comment]
Hi @casoetan, thanks for raising this, I'll look into setting a default `BASE_DIR` setting within the `django-drf-filepond` app so that it's not a requirement to have `BASE_DIR` defined in your top-level settings.

**Comment by jcohen02** [comment]
@casoetan I've just merged #10 into master to fix this issue, the fix will be included in the next release.
